rockhound/slab2.py

```diff
@@ -0,0 +1,141 @@
+            - `alaska`: Alaska
+            - `calabria`: Calabria
+            - `caribbean`: Caribbean
+            - `cascadia`: Cascadia
+            - `central_america`: Central America
+            - `cotabalo`: Cotabalo
+            - `halmahera`: Halmahera
+            - `hellenic`: Hellenic Arc
+            - `himalaya`: Himalaya
+            - `hindu_kush`: Hindu Kush
+            - `izu_bonin`: Izu-Bonin
+            - `kamchatka`: Kamchatka-Kuril Islands-Japan
+            - `kermadec`: Kermadec
+            - `makran`: Makran
+            - `manila_trench`: Manila Trench
+            - `muertos_trough`: Muertos Trough
+            - `new_guinea`: New Guinea
+            - `pamir`: Pamir
+            - `philippines`: Philippines
+            - `puysegur`: Puysegur
+            - `ryukyu`: Ryukyu
+            - `scotia_sea`: Scotia Sea
+            - `solomon_islands`: Solomon Islands
+            - `south_america`: South America
+            - `sulawesi`: Sulawesi
+            - `sumatra_java`: Sumatra-Java
+            - `vanuatu`: Vanuatu
+    load : bool
+        Wether to load the data into an :class:`xarray.Dataset` or just return the
+        path to the downloaded data. If False, will return a list with the paths to the
+        subduction grids, respectively.
+    kwargs
+        Keyword arguments will be forwarded to the :func:`xarray.open_dataset` function
+        that loads the grid into memory.
+
+    Returns
+    -------
+    grid : :class:`xarray.Dataset` or str
+        The loaded grid or the file path to the downloaded data.
+    """
+    if zone not in ZONES:
+        raise ValueError("Invalid slab zone: {}".format(zone))
+    fnames = [
+        REGISTRY.fetch(
+            "{}_slab2_{}.grd".format(ZONES[zone]["fname_indicator"], dataset)
+        )
+        for dataset in DATASETS
+    ]
+    if not load:
+        return fnames
+    arrays = [xr.open_dataarray(f).rename(x="longitude", y="latitude") for f in fnames]
+    for array, dataset in zip(arrays, DATASETS):
+        array.name = dataset
+        # Change long_name and add units of each array
+        array.attrs["long_name"] = DATASETS[dataset]["name"]
+        array.attrs["units"] = DATASETS[dataset]["units"]
+    # Merge arrays into a single xr.Dataset
+    ds = xr.merge(arrays)
+    # Change units of thickness, depth and depth_uncertainty to meters
+    ds["thickness"] *= 1000
+    ds["depth"] *= 1000
+    ds["depth_uncertainty"] *= 1000
+    # Change long_name and units of longitude and latitude coords
+    ds.longitude.attrs["long_name"] = "Longitude"
+    ds.longitude.attrs["units"] = "degrees"
+    ds.latitude.attrs["long_name"] = "Latitude"
+    ds.latitude.attrs["units"] = "degrees"
+    # Add attributes to the xr.Dataset
+    ds.attrs.update(
+        {
+            "title": "Slab2 model - Zone: {}".format(ZONES[zone]["name"]),
+            "zone": ZONES[zone]["name"],
+            "datum": "WGS84",
+            "doi": "10.5066/F7PV6JNV.",
```

Comment:
Remove the final dot:
```suggestion
            "doi": "10.5066/F7PV6JNV",
```